docs/debugging.md

```diff
@@ -0,0 +1,90 @@
+# Debugging
+
+## Connecting to VMs Using SSH/RDP
+- To debug Linux VMs, SSH can be used to login to the machines for troubleshooting and viewing log files. 
+- To debug Windows VMs, a RDP client such as Windows Remote Desktop on Windows or xfreerdp on Linux can be used to login to the machines for troubleshooting and viewing log files.
+- Workstation VMs are not exposed to the internet and do not have public IPs, a bastion host such as the DC or Connector can be used to access the Workstation VMs on the private network.
+
+### Connecting to CentOS Workstations
+One way to access CentOS Workstations is to use the Connector as a bastion host. Execute the following command to first SSH into the Connector:
+
+```ssh -A <login-user>@<cac-public-ip>``` 
+
+From inside the Connector, execute the following command to SSH into the CentOS Workstation:
+
+```ssh <login-user>@<centos-internal-ip>```
+
+### Connecting to Windows Workstations
+One way to access Windows Workstations is to use the Domain Controller as a bastion host. Use the following credentials to connect to the DC using a RDP client:
+
+```
+Computer: <domain-controller-public-ip>
+User: <login-user>
+Password: <dc_admin_password>
+```
+
+From inside the Domain Controller, connect to the Windows Workstation by using Windows RDP and entering the following login credentials:
+
+```
+Computer: <win-internal-ip>
+User: <login-user>
+Password: <dc_admin_password>
+```
+
+## VM Log Locations
+
+### Amazon Web Services
+
+| VM Instance      | Login User    | Log File Path                               | Description                                                 |
+| :--------------- | :------------ | :------------------------------------------ | :---------------------------------------------------------- |
+| cac              | ubuntu        | /var/log/syslog                             | Detailed system log for startup and provisioning            |
+|                  |               | /var/log/teradici/provisioning.log          | Minimal log for provisioning script                         |
```

Comment:
is it still minimal log? thought we added set -x to show everything?

Reply:
That commit (improve provisioning scripts branch) hasn't been merged yet, but I've changed this table to show detailed logs instead so I don't have to go back to it later.